app/controllers/public.js

```diff
@@ -37,4 +37,14 @@ export default class PublicController extends Controller {
   toggleMenu() {
     this.toggleProperty('isMenuOpen');
   }
+
+  @action
+  goToLink(section) {
+      this.transitionToRoute('public.index');
+      setTimeout(function(){       
```

Comment:
This will most likely fail if the transition is not completed within the timeout

Reply:
Is their other way so that function gets executed only when the page is loaded?